### PR TITLE
fix travis

### PR DESCRIPTION
**Pull request by Mathieudewet**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -54,32 +52,24 @@ script:
...
+    bucket: api-platform-demo
+    skip_cleanup: true
+    acl: public-read
+    local-dir: admin/build
+    on:
+      branch: master
+      repo: api-platform/demo
+
+after_deploy:
+    ./script/after_deploy.sh
```
Comment (dunglas) [inline .travis.yml]: As this script is a one liner, you can replace this line by its content and remove the script

File: script/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+gcloud docker -- push eu.gcr.io/${PROJECT_NAME}/varnish:latest
+helm install --wait --namespace=${TRAVIS_COMMIT} ./api/helm/api --set php.repository=eu.gcr.io/${PROJECT_NAME}/php --set nginx.repository=eu.gcr.io/${PROJECT_NAME}/nginx --set varnish.repository=eu.gcr.io/${PROJECT_NAME}/varnish --set secret=${APP_SECRET} --set postgresUser=${DATABASE_USER},postgresPassword="${DATABASE_PASSWORD}",postgresDatabase=${DATABASE_NAME} --set postgresql.persistence.enabled=true ${IP}
+sleep 60
+kubectl exec -it $(kubectl --namespace=${TRAVIS_COMMIT} get pods -l app=api-php -o jsonpath="{.items[0].metadata.name}") --namespace=${TRAVIS_COMMIT} -- ash -c'export APP_ENV=dev && composer install -n && bin/console d:s:u --force --env=dev && bin/console hautelook:fixtures:load -n && APP_ENV=prod composer --no-dev install --classmap-authoritative && bin/console d:s:u --env=prod'
+kubectl label namespace ${TRAVIS_COMMIT} app=api-demo
+export REACT_APP_API_ENTRYPOINT_IP=$(kubectl --namespace `echo ${TRAVIS_COMMIT}` get ingress -o jsonpath='{.items[0].status.loadBalancer.ingress[0].ip}')
+cd admin && yarn && REACT_APP_API_ENTRYPOINT=http://${REACT_APP_API_ENTRYPOINT_IP} yarn build --environment=prod
```
Comment (dunglas) [inline script/deploy.sh]: too late

Comment (dunglas) [reply script/deploy.sh]: it should use HTTPS, and mainly, use the domain name, not the IP

File: script/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+helm dependencies update ./api/helm/api
+docker build --pull -t eu.gcr.io/${PROJECT_ID}/php -t eu.gcr.io/${PROJECT_ID}/php:latest api --target api_platform_php
+docker build --pull -t eu.gcr.io/${PROJECT_ID}/nginx -t eu.gcr.io/${PROJECT_ID}/nginx:latest api --target api_platform_nginx
+docker build --pull -t eu.gcr.io/${PROJECT_ID}/varnish -t eu.gcr.io/${PROJECT_ID}/varnish:latest api --target api_platform_varnish
+gcloud docker -- push eu.gcr.io/${PROJECT_ID}/php:latest
+gcloud docker -- push eu.gcr.io/${PROJECT_ID}/nginx:latest
+gcloud docker -- push eu.gcr.io/${PROJECT_ID}/varnish:latest
+#TODO use rolling update
+helm install --wait --namespace=${TRAVIS_COMMIT} ./api/helm/api --set php.repository=eu.gcr.io/${PROJECT_ID}/php --set nginx.repository=eu.gcr.io/${PROJECT_ID}/nginx --set varnish.repository=eu.gcr.io/${PROJECT_ID}/varnish --set secret=${APP_SECRET} --set postgresUser=${DATABASE_USER},postgresPassword="${DATABASE_PASSWORD}",postgresDatabase=${DATABASE_NAME} --set postgresql.persistence.enabled=true --set ingress.annotations.kubernetes.io/ingress.global-static-ip-name: api-platform-demo-ip
+sleep 60
```
Comment (Simperfit) [inline script/deploy.sh]: you don't need to wait, just check that the pods is running and do a while until it is

**Pull request merged**